api/graphql/schema.graphql

```diff
@@ -318,7 +318,7 @@ type Pool {
   classifications: [Classification] @belongsToMany
   operationalRequirements: [OperationalRequirement]
     @rename(attribute: "operational_requirements")
-  poolCandidates: [PoolCandidate] @hasMany @can(ability: "viewAny")
+  poolCandidates: [PoolCandidate] @hasMany @can(ability: "view", resolved: true)
```

Comment:
Okay, I added the scope and while I was it it I think I noticed that the view policy could have resulted in some bugs so I rearranged it to avoid letting people through too easily or restricting them too much. Hopefully, I got it right this time 😅 

Fixed in [refactor view policy for pool candidate](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5815/commits/0f5c99915c3a16003ad09aae8fcaf08087bb5cb6) then updated tests in [fix pool candidate count test](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5815/commits/2e1f1cb51ad5ee3b9be6b10960e2f955d8bdaaea)

Reply:
Sadly, not possible since `@scope` cannot be used on a field definition 😢